rubicon_ml/sklearn/pipeline.py

```diff
@@ -147,18 +156,24 @@ def score(self, X, y=None, sample_weight=None, experiment=None):
         float
             Result of calling `score` on the final estimator.
         """
-        score = super().score(X, y, sample_weight)
+        with warnings.catch_warnings():
+            if self.ignore_warnings:
+                warnings.simplefilter("ignore")
+            score = super().score(X, y, sample_weight)
 
-        if experiment is not None:
-            self.experiment = experiment
-        elif self.experiment is None:
-            self.experiment = self.project.log_experiment(**self.experiment_kwargs)
+            if self.ignore_warnings:
+                warnings.simplefilter("ignore")
```

Comment:
do we need to do this twice in this function?

Reply:
Thanks for catching this! No we do not- I had this here when initially considering where to place the `ignore_warnings` check and forgot about it when ultimately deciding to place it within the context manager at the top. Removing now.